src/vulkan/gpu.c

```diff
@@ -220,10 +220,14 @@ static pl_handle_caps vk_sync_handle_caps(struct vk_ctx *vk)
 {
     pl_handle_caps ret = 0;
 
-#if VK_HAVE_UNIX
+#ifdef VK_HAVE_UNIX
     if (vk->vkGetSemaphoreFdKHR)
         ret |= PL_HANDLE_FD;
 #endif
+#ifdef VK_HAVE_WIN32
+    if (vk->vkGetSemaphoreWin32HandleKHR)
+        ret |= (PL_HANDLE_WIN32 | PL_HANDLE_WIN32_KMT)
```

Comment:
missing semicolon

Reply:
If the right handle to use depends on the windows version, then shouldn't we export the cap based on the windows version as well?